hydrosdk/model.py

```diff
@@ -324,6 +340,42 @@ def __repr__(self):
         return "Model {}:{}".format(self.name, self.version)
 
 
+class ExternalModel:
+    BASE_URL = "/api/v2/externalmodel"
+
+    @staticmethod
+    def ext_model_json_to_ext_model(ext_model_json: dict):
+        return ExternalModel(name=ext_model_json["model"]["name"],
+                             id_=ext_model_json["model"]["id"],
+                             contract=contract_from_dict(ext_model_json["modelContract"]),
+                             metadata=ext_model_json.get("metadata"), version=ext_model_json["modelVersion"])
+
+    @staticmethod
+    def create(cluster, ext_model: dict):
```

Comment:
why dict? create a class or add required fields to function signature

Reply:
Fixed